Identity/BusinessLayer/DTOs/BlockingDto.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
```

Comment:
you throw exceptions but i dont see you to handle them anywhere in your code. create global exception middleware